### PR TITLE
feature/species-model

### PR DESCRIPTION
**Pull request by brzeczkowskaw**

### CODE REVIEW

**Review by Suegro24 — CHANGES_REQUESTED** [review]
I think that name should be required and unique.

**Review by Suegro24 — APPROVED** [review]

**Review by kami3la — APPROVED** [review]

**Review by Suegro24 — COMMENTED** [review]
import from file `example` is pointless because we will delete this file as soon as possible
try to name models and schemas in singular

File: server/models/example.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const mongoose = require('mongoose');
+export const mongoose = require('mongoose');
```
Comment (jedrzejratajczak) [inline server/models/example.ts]: imo we should export mongoose in `server.ts`

File: server/models/spieces.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    name: {
+        type: String,
+        required: true,
+        unique: true
+    }
+})
+
+const Species = mongoose.model('specie', SpeciesSchema);
+
+module.exports = Species
```
Comment (jedrzejratajczak) [inline server/models/spieces.ts]: we should use ES6 or CJS modules, not both
i would suggest using ES6

File: server/models/spieces.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const SpeciesSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        unique: true
+    }
+})
+
+const Species = mongoose.model('specie', SpeciesSchema);
```
Comment (jedrzejratajczak) [inline server/models/spieces.ts]: `specie`

**Review by jedrzejratajczak — CHANGES_REQUESTED** [review]

**Review by jedrzejratajczak — APPROVED** [review]

**Pull request merged**